front/admin/cli.ts

```diff
@@ -271,6 +272,17 @@ const workspace = async (command: string, args: parseArgs.ParsedArgs) => {
       return;
     }
 
+    case "classify-messages": {
```

Comment:
Does it need to be in the cli? Shouldn't it be a standalone script?

Reply:
added a CLI using makeScript() 👍

Reply:
Should we remove it from here then :p?